windows_create_package.bat

```diff
@@ -26,15 +26,23 @@ md dist\windows\ovms
 copy bazel-bin\src\ovms.exe dist\windows\ovms
 if !errorlevel! neq 0 exit /b !errorlevel!
 
-copy  %cd%\bazel-out\x64_windows-opt\bin\src\python39.dll dist\windows\ovms
-if !errorlevel! neq 0 exit /b !errorlevel!
-
 copy  %cd%\bazel-out\x64_windows-opt\bin\src\python\binding\pyovms.pyd dist\windows\ovms
 if !errorlevel! neq 0 exit /b !errorlevel!
 
 copy C:\%output_user_root%\openvino\runtime\bin\intel64\Release\*.dll dist\windows\ovms
 if !errorlevel! neq 0 exit /b !errorlevel!
 
+:: Prepare self-contained python
+set "dest_dir=C:\opt"
+set "python_version=3.9.13"
+call %cd%\windows_prepare_python.bat %dest_dir% %python_version%
+:: Copy whole catalog to dist folder and install dependencies required by LLM pipelines
+xcopy %dest_dir%\python-%python_version%-embed-amd64 dist\windows\ovms\python /E /I /H
+.\dist\windows\ovms\python\python.exe -m pip install "Jinja2==3.1.4" "MarkupSafe==3.0.2"
+if !errorlevel! neq 0 (
+    echo Error copying python into the distribution location. The package will not contain self-contained python.
+)
+
```

Comment:
we should included also the licenses in the package - for ovms and all 3rdparty...

Reply:
This PR blocks further testing and samples adjustments. Let's include licenses in another PR.